revscoring/scoring/models/util.py

```diff
@@ -0,0 +1,71 @@
+import json
+
+import numpy
+
+
+def normalize(v):
```

Comment:
I looked into that.  I spent hours on it.  IMO, it's a total mess that I was happy to write these functions to replace.